web3/_utils/module_testing/eth_module.py

```diff
@@ -2807,18 +2807,20 @@ def test_eth_getBlockTransactionCountByNumber_block_with_txn(
     def test_eth_getUncleCountByBlockHash(
         self, w3: "Web3", empty_block: BlockData
     ) -> None:
-        uncle_count = w3.eth.get_uncle_count(empty_block["hash"])
+        with pytest.warns(DeprecationWarning):
```

Comment:
We like to match on the message for deprecation warnings, so this would just need a `match=r"All get_uncle\* methods have been deprecated"`